rust-code-analysis-cli/src/web/server.rs

```diff
@@ -626,64 +626,74 @@ mod tests {
-                                               "mi": {"mi_original": 151.433_315_883_223_23,
-                                                      "mi_sei": 142.873_061_717_489_78,
-                                                      "mi_visual_studio": 88.557_494_668_551_6}},
-                                   "name": "foo",
-                                   "spaces": []}]}
-        });
+                    "id": "1234",
+                    "language": "python",
+                    "spaces": {"kind": "unit",
+                               "start_line": 1,
+                               "end_line": 4,
+                               "metrics": {"cyclomatic": {"sum": 2.0, "average": 1.0},
+                                           "cognitive": {"sum": 0.0, "average": 0.0},
+                                           "nargs": {"sum": 0.0, "average": 0.0},
+                                           "nexits": {"sum": 0.0, "average": 0.0},
+                                           "halstead": {"bugs": 0.000_942_552_557_372_941_4,
+                                                        "difficulty": 1.0,
+                                                        "effort": 4.754_887_502_163_468,
+                                                        "length": 3.0,
+                                                        "estimated_program_length": 2.0,
+                                                        "purity_ratio": 0.666_666_666_666_666_6,
+                                                        "level": 1.0,
+                                                        "N2": 1.0,
+                                                        "N1": 2.0,
+                                                        "vocabulary": 3.0,
+                                                        "time": 0.264_160_416_786_859_36,
+                                                        "n2": 1.0,
+                                                        "n1": 2.0,
+                                                        "volume": 4.754_887_502_163_468},
+                                           "loc": {"cloc": 1.0, "ploc": 2.0, "lloc": 1.0, "sloc": 4.0, "blank": 1.0},
+                                           "nom": {"functions": 1.0, "closures": 0.0, "total": 1.0},
+                                           "mi": {"mi_original": 139.974_331_558_152_1,
+                                                  "mi_sei": 161.414_455_240_662_22,
+                                                  "mi_visual_studio": 81.856_334_244_533_39}},
+                               "name": "test.py",
+                               "spaces": [{"kind": "function",
+                                           "start_line": 3,
+                                           "end_line": 4,
+                                           "metrics": {"cyclomatic": {"sum": 1.0, "average": 1.0},
+        <<<<<<< HEAD
```

Comment:
Something went wrong during the rebase :)

Reply:
Sorry, I'm tired :P